lib/paypal_nvp.rb

```diff
@@ -49,7 +60,7 @@ def call_paypal(data)
       http.ca_path = @rootCA
       http.verify_mode = OpenSSL::SSL::VERIFY_PEER
       http.verify_depth = 5
-    elsif File.exists?(@rootCA)
+    elsif File.exist?(@rootCA)
```

Comment:
Good spot @jaxi :clap: I didn't realise `File.exists?` is now deprecated in favour of `File.exist?`. TIL :+1: